wordpress-sites-worldwide.md

```diff
@@ -40,6 +38,11 @@ Please submit changes and additions to this repository via Patch or Pull Request
 - [Reuters Blogs](http://blogs.reuters.com/us/)
 - [The Sun](https://www.thesun.co.uk/) (Leading UK tabloid newspaper)
 
+## Music
+- [Alanis Morissette](http://alanis.com/)
+- [Alicia Keys](http://aliciakeys.com/)
+- [Beyoncé](http://www.beyonce.com/)
```

Comment:
I don't think the Beyoncé site is particularly very good. It was created in 2014.